src/main/java/seedu/meetingjio/timetables/MasterTimetable.java

```diff
@@ -5,7 +5,7 @@
 import java.util.ArrayList;
 
 public class MasterTimetable {
-    private final ArrayList<Timetable> timetables;
+    public static ArrayList<Timetable> timetables;
```

Comment:
Better to make the attributes private so that it is more secured unless there's a need to access it publicly

Reply:
Agreed but listCommand needs public access to it. Good observation though!